From 2c4f64ce33f86186bcc3d8501b5498dd54800bc0 Mon Sep 17 00:00:00 2001
From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Thu, 4 Feb 2016 18:59:19 +0100
Subject: [PATCH 282/365] net: dev: make xmit_rec_dec() void

Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
---
 net/core/dev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/core/dev.c b/net/core/dev.c
index 0aaf93b..14f7b58 100644
--- a/net/core/dev.c
+++ b/net/core/dev.c
@@ -2984,7 +2984,7 @@ static inline void xmit_rec_inc(void)
 	__this_cpu_inc(xmit_recursion);
 }
 
-static inline int xmit_rec_dec(void)
+static inline void xmit_rec_dec(void)
 {
 	__this_cpu_dec(xmit_recursion);
 }
-- 
2.7.4

